assets/scss/application.scss

```diff
@@ -4,9 +4,8 @@ $path: "/assets/images/";
 $moj-page-width: 1170px;
 $govuk-page-width: $moj-page-width;
 
-@import 'govuk/all';
-@import 'moj/all';
-@import 'hmpps/all';
+@import '../../node_modules/govuk-frontend/dist/govuk/all';
```

Comment:
This seems like a backward step. Shouldn't we be importing node_modules in to the SASS compiler, so it would be something like govuk-frontend/dist/govuk/all?